vcpkg.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "microsoft-ifc-sdk",
   "version-semver": "0.43.1",
```

Comment:
I forgot to mention this: can you change the version to 0.43.2?  Same in the top-level CMakeLists.txt.

Otherwise, every else looks good.  Thanks for your patience!

Reply:
I would prefer to do this in a separate commit to keep changes more isolated.

Reply:
> I would prefer to do this in a separate commit to keep changes more isolated.

OK, int that case, let's commit that other patch first.

Reply:
Done: #57